src/style.css

```diff
@@ -0,0 +1,17 @@
+
+body {
+  background-color: #000;
+  margin: 43px;
+}
+
+.block {
+  width: 284px;
+  margin-bottom: 86px;
+  padding: 10px;
+  background-color: #ddd;
+  box-shadow: 0 0 0 1px #e75551,
+    0 0 0 11px #ffa500,
+    0 0 0 12px #e75551,
+    0 0 0 22px #308dc0,
+    0 0 0 23px #e75551;
```

Comment:
instead of this complicated box-shadow use three nested divs, like this:
```
<div class='block color1'>
  <div class='block color2'>
    < div class='block color2'>
```
and add needed styles to the needed class